ocaml/xapi-consts/api_errors.ml

```diff
@@ -591,3 +591,4 @@ let cluster_does_not_have_one_node = "CLUSTER_DOES_NOT_HAVE_ONE_NODE"
 let no_compatible_cluster_host = "NO_COMPATIBLE_CLUSTER_HOST"
 let cluster_force_destroy_failed = "CLUSTER_FORCE_DESTROY_FAILED"
 let clustering_enabled_on_network = "CLUSTERING_ENABLED_ON_NETWORK"
+let cluster_stack_in_use = "CLUSTER_STACK_IN_USE"
```

Comment:
We should define this in `ocaml/idl/datamodel_errors.ml` too

Reply:
Whoops, forgot that